pycaching/cache.py

```diff
@@ -853,7 +878,12 @@ def load_quick(self):
         # prettify data
         self.name = data["name"]
         self.type = Type.from_string(data["type"]["text"])
-        self.state = data["available"]
+
+        if data["available"]:
+            self.status = Status.enabled
+        else:
+            self.status = Status.archived if data["archived"] else Status.disabled
```

Comment:
If we omit these two lines, full lazy load is triggered when user accesses `.status` -> filling the `.status` correctly. Right?

Reply:
Good point 👍
Fixed in https://github.com/tomasbedrich/pycaching/pull/193/commits/43b3d3db9efae54cb47bd500709ad89999b5ed74